packages/create-onchain/templates/minikit-basic/app/globals.css

```diff
@@ -1,25 +1,32 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
+@custom-variant dark (&:where(.dark, .dark *));
```

Comment:
follow-up: We should consider scoping this or using data-attributes for this. This technically leaks OnchainKit internals to the parent, which we should avoid if we can figure out a good way to do so